.github/workflows/check.yml

```
@@ -0,0 +1,31 @@
+name: "check"
+on: [push, pull_request]
+
+jobs:
+  assert-formatted:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
```

question: do we care about the specific version?
```suggestion
          node-version: lts
```

---

I was matching with what's in `tool-versions` but I'm fine with this.

---

Oh I didn't realize it was there: looks like you can use `node-version-file` to get it from there: https://github.com/actions/setup-node/blob/main/action.yml#L10-L11